config/database.pg.yml

```diff
@@ -15,7 +15,7 @@ base: &base
   adapter: postgresql
   encoding: utf8
   username: root
-  pool: 1
+  pool: 5

```

Comment:
I'm ok with changing this as long as rails times out unused connections.  Each pg postmaster client processes on appliances consume around 100 MB of memory or more so we need to ensure they're actually being used.  We can configure the database to time them out but the client should really not be wasting connections.